ui/raidboss/data/triggers/o11s.js

```diff
@@ -85,11 +158,11 @@
       alertText: function(data) {
         if (data.lastWasStarboard) {
           return {
-            en: 'Move (Left)',
+            en: 'Move',
```

Comment:
Personally, I like the "left" in here.  I feel like folks have been very split on what they want here, whether they want "move" or "left" called out, and so I really like having both.